applications/nrf5340_audio/src/bluetooth/le_audio_bis_headset.c

```diff
@@ -270,6 +270,22 @@ static void pa_sync_lost_cb(struct bt_bap_broadcast_sink *sink)
 	}
 }
 
+static int bis_specific_codec_config(struct bt_bap_base_bis_data bis_data,
```

Comment:
It is really hard to understand what this function does.
Maybe add function documentation?